test/fixtures/rules/css-modules/unused.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import badStyles from './test.css';
+
```

Comment:
Can you remove the empty line between the imports?

Reply:
ok changing right now